src/applications/healthcare/questionnaire/tests/e2e/01.questionnaire.demographics.cypress.spec.js

```diff
@@ -0,0 +1,71 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import basicUser from './fixtures/users/user-basic.json';
+
+describe('healthcare questionnaire -- demographics', () => {
+  before(() => {
+    cy.server();
+    cy.fixture(
```

Comment:
We have a [custom command for fixtures](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-migration-guide.md#test-data-fixtures-) if you'd like to just import all fixtures from your local app directory.

Reply:
Good to know for when the test get more complex